File: library-grad-project/Models/Book.cs
using System.Collections.Generic;

namespace LibraryGradProject.Models
{
    public class Book
    {
        public int BookId { get; set; }
        public string ISBN { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public string PublishDate { get; set; }

        // One to many
        public ICollection<BookReservation> BookReservations { get; set; }
    }
}

File: client-side/src/public/app/shared/book.ts
export class Book {
    BookId: number;
    // ISBN: string;
    // Title: string;
    // Author: string;
    // PublishDate: string;

    constructor(
        public ISBN: string,
        public Title: string,
        public Author: string,
        public PublishDate: string
    ){ }
}
